recipe/meta.yaml

```
@@ -30,6 +30,7 @@ test:
 about:
   home: https://github.com/benediktschmitt/py-filelock
   license: Public Domain
```

Could you please fix the ``license`` as well?
I think it should be
```suggestion
  license: Unlicense
```

---

https://spdx.org/licenses/Unlicense.html

---

Thanks!